conf/application-context.xml

```diff
@@ -368,41 +358,37 @@
     </bean>
 
     <bean id="authenticationService" class="org.sagebionetworks.bridge.services.AuthenticationServiceImpl">
-        <property name="stormpathClient" ref="stormpathClient" />
         <property name="distributedLockDao" ref="distributedLockDao"/>
         <property name="cacheProvider" ref="cacheProvider" />
         <property name="bridgeConfig" ref="bridgeConfig" />
-        <property name="accountEncryptionService" ref="accountEncryptionService" />
+        <property name="accountDao" ref="accountDao" />
         <property name="consentService" ref="consentService" />
         <property name="optionsService" ref="optionsService" />
+        <property name="healthCodeService" ref="healthCodeService" />
         <property name="signInValidator">
             <bean class="org.sagebionetworks.bridge.validators.SignInValidator"/>
         </property>
+        <property name="signUpValidator">
+            <bean class="org.sagebionetworks.bridge.validators.SignUpValidator"/>
+        </property>
```

Comment:
[MINOR] Is there any value in making this a bean? It's just a bunch of simple null checks and empty string checks, so we're unlikely to ever need a separate implementation or a mock. Why not just create a static instance object and call SignUpValidator.INSTANCE?